roles/network_plugin/cilium/templates/cilium/config.yml.j2

```diff
@@ -69,6 +69,13 @@ data:
   # custom-cni-conf to "true", otherwise Cilium may overwrite the configuration.
   custom-cni-conf: "false"
 
+{% if cilium_version | regex_replace('v') is version('1.14.0', '>=') %}
+  # Tell the agent to generate and write a CNI configuration file
+  write-cni-conf-when-ready: /etc/cni/net.d/05-cilium.conflist
+  cni-exclusive: "{{ cilium_cni_exclusive }}"
```

Comment:
> This will also fix the fact that the keys cilium_cni_exclusive and cilium_cni_log_file are no longer used when deploying cilium 1.14.0 .

These only be used when cilium version < 1.14.0, right?

Reply:
No these are still needed when cilium version >= 1.14.0 . They deleted the `cni-install.sh` used in the postStart hook which was taking these keys as args but now they need to be passed via the ConfigMap.

Reply:
Works great, did need to add the 05-cilium.conflist to /etc/cni/net.d/ because the task is deploying the 000-cilium-portmap.conflist, do we want to include another if statement in the task for this?